compiler/daml-extension/src/extension.ts

```diff
@@ -406,6 +410,21 @@ namespace DamlVirtualResourceNoteNotification {
   );
 }
 
+interface VirtualResourceFormattingStartedParams {
+  /** The virtual resource uri */
+  uri: string;
+
+  /** A message describing the formatting */
+  message: string;
```

Comment:
this doesn't seem to be used

Reply:
You're right, this was meant for messaging relating to https://github.com/digital-asset/daml/issues/15782, I'll remove it from this PR for now